### ISSUE TITLE
Unable to create sorting disposal pipes

### ISSUE THREAD

**Issue by Muhdir** [ParadiseSS13/Paradise#8932]
**Problem Description**:
Ever had a bomb blow all your pipes away? Felt ambitious enough to try to fix all that? Well, if the sorting disposal pipes are ever broken you won't be able to create them by any means with the RPD or the not so mobile disposal pipe dispenser. You can try to fill them in by the regular junctions, but that will mean that depending on the type of junction, either deliveries will never make it to that department or that everything through that pipe will end up in that department.

This is the pipe in question
![disposals](https://user-images.githubusercontent.com/38748088/39326801-625a5ce0-4996-11e8-9937-87e501ab44d1.png)

**What did you expect to happen**: 
I would've been able to create those disposal pipe sections. And with an destination tagger repaired the delivery system.

**What happened instead**:
I neither was able to make sorting disposal pipes nor do I start with a destination tagger to set the destination of that pipe.

**Why is this bad/What are the consequences:**
If someone either removes, destroys or displaces the sorting pipe, disposals deliveries will cease to function properly depending where that section was removed. And if you are a clown, removing and placing the sorting pipe unsets the destination and will force everything out of that departments disposal bin. 

**Steps to reproduce the problem**:
Destroy a sorting disposal pipe then scream to everyone that disposals deliveries are fucked and shouldn't be used anymore

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Anything else you can tell us.

Repairing pipes is obviously not done on the short gamemodes, but traitors can obviously hijack crates with less suspicion if they have these tools available to them.

**Comment by Citinited** [comment]
I think (I could be wrong) disposals needs an entire overhaul before you can begin to think of dispensing sort junctions, as they're sort of hardcoded into the game right now - so while it is an issue, it would take a refactor to fix.

**Comment by Fox-McCloud** [comment]
```
I think (I could be wrong) disposals needs an entire overhaul before you can begin to think of dispensing sort junctions, as they're sort of hardcoded into the game right now - so while it is an issue, it would take a refactor to fix.
```

This. Disposals code is utterly *horrifying*.

**Comment by Citinited** [comment]
Right, so I took a look at this and actually it does seem like you could just make the RPD dispense a sortjunction pipe and then use the tagger to change destination, like you said. You won't be able to do anything like add new destinations to the sortjunction though. I'll look into adding it to the RPD, once the refactor goes through.

**Comment by Citinited** [comment]
I'll look into it after work and see if it's possible/feasible to add this to the RPD. I doubt you can have much added functionality beyond replacing destroyed pipes though
